apps/microsoft-teams/frontend/src/components/config/ConfigPage/ConfigPage.spec.tsx

```diff
@@ -1,18 +1,23 @@
 import ConfigPage from './ConfigPage';
 import { describe, expect, it, vi } from 'vitest';
 import { render, screen } from '@testing-library/react';
-import { accessSection, notificationsSection } from '@constants/configCopy';
+import { accessSection } from '@constants/configCopy';
 import { mockSdk } from '@test/mocks';
 
 vi.mock('@contentful/react-apps-toolkit', () => ({
   useSDK: () => mockSdk,
 }));
 
+vi.mock('@components/config/NotificationsSection/NotificationsSection', () => ({
```

Comment:
I added a mock for this child component because I was seeing errors from the NotificationsSection component when it calls our hooks for getting the content types and channels. These errors were only happening when this test was running in CI.